### PR TITLE
Amend README.md

### PR DESCRIPTION
**Pull request by kfate**
Fixed the incorrect information with README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,26 @@ This app will help you touch up any section of a photo as you draw on it. This c
 
 ## Getting Started
 
-Clone the project in your local machine, and run the program using
-```bash
-python3 qt_touchup_app.py
-```
+Clone the project in your local machine, and run the program using a command that will be specified later
```
Comment (jryzkns) [inline README.md:7]: the command to run the project is 

```bash
python3 main.py
```

File: README.md [hunk 2]
```diff
@@ -4,27 +4,26 @@ This app will help you touch up any section of a photo as you draw on it. This c
...
 
-This application requires all of the following:
+As of now, this application requires all of the following:
 
 ```
 python3
 PyQt5
 numpy
 PIL
+OpenCv3
```
Comment (jryzkns) [inline README.md:18]: idk where you found the version 3 but what should go here is `opencv-python`

File: README.md [hunk 3]
```diff
@@ -4,27 +4,26 @@ This app will help you touch up any section of a photo as you draw on it. This c
...
 ```
 
 ## Built With
 
 * [PyQt5](https://www.riverbankcomputing.com/software/pyqt/) - The GUI framework used
 * [Numpy](https://numpy.org/) - Used for calculations
 * [PIL](http://www.pythonware.com/products/pil/) - Python Image Library
+* [OpenCv](https://opencv.org/)- Library of programming functions mainly aimed at real-time computer vision
```
Comment (jryzkns) [inline README.md:26]: OpenCV

File: README.md [hunk 4]
```diff
@@ -4,27 +4,26 @@ This app will help you touch up any section of a photo as you draw on it. This c
 
 ## Getting Started
```
Comment (jryzkns) [inline README.md:5]: impuritiers is not the correct spelling, you can can blemishes instead

**Review by jryzkns — CHANGES_REQUESTED** [review]
Please address the comments then squash all the commits together in one commit. Pick an appropriate commit message when you squash.

**Comment by kfate** [comment]
The suggested changes have been made.

**Pull request merged**